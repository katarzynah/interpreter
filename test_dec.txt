program f;
var i, j, k, l: int;
s: string;
b : bool;

begin
end.

