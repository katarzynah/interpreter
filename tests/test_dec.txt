program f;
var i, j, k, l: int;
s: string;
b : bool;
/*
procedure hej(x : int;);
var y: string;
begin
print(y);
print("hej");
end;
*/
begin
end.

