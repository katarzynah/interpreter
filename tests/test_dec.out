"k =" 3
"j = k + 2 =" 5
"i = j * k =" 15
"l = i / k =" 5
"b := l = j =" True
"k = 12 - l =" 7
[(fromList [(Ident "b",True),(Ident "i",15),(Ident "j",5),(Ident "k",7),(Ident "l",5),(Ident "s","haj")],fromList [])]
